backend/app/api/v1/endpoints/files.py

```
@@ -0,0 +1,21 @@
+@router.post("/add")
+@auth_required()
+async def add_file(request: Request, body: AddFileRequest):
+    current_user = request.state.user
+    
+    user_file = FileService().add_file_to_db(user_id=current_user.id, file_type=body.fileType, file_name=body.fileName, file_key=body.fileKey, file_size=body.fileSize)
+    return ResponseHandler().success_response(data=user_file, message="added file to database successfully")
+
+@router.get("/get")
+@auth_required()
+async def get_file(request: Request, body: GetFileRequest):
+    current_user = request.state.user
```

Ruff (0.14.10)

Local variable `current_user` is assigned to but never used

Remove assignment to unused variable `current_user`

(F841)